client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,38 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+/**
+ * Adds the ability to trigger a button click event using a keyboard shortcut
+ * declared on the controlled element.
+ *
+ * @example
+ * ```html
+ * <button type="button" data-controller="w-kbd" data-w-kbd-key="[">Trigger me with the <kbd>[</kbd> key.</button>
+ * ```
+ */
+export class KeyboardController extends Controller<HTMLButtonElement> {
+  static values = { key: { default: '', type: String } };
+
+  handleKey(event: Event) {
+    if (event.preventDefault) event.preventDefault();
+    this.element.click();
+  }
+
+  initialize() {
+    this.handleKey = this.handleKey.bind(this);
+    if (!this.keyValue && this.element.ariaKeyShortcuts) {
+        this.keyValue = this.element.ariaKeyShortcuts;
+    }
+    if (!this.keyValue && this.element.ariaKeyShortcuts) {
+        this.keyValue = this.element.ariaKeyShortcuts;
+    }
+  }
+
+  keyValueChanged(key, previousKey) {
```

Comment:
```suggestion
  /**
  * When a key is set or changed, bind the handler to the keyboard shortcut. This will override the shortcut, if already set, ... Link to docs
  */
  keyValueChanged(key, previousKey) {
```


Something like this.